conda-environment.yml

```diff
@@ -58,6 +58,7 @@ dependencies:
   - flake8==2.6.0
   - mccabe==0.5.0
   - mock==2.0.0
+  - numpy==1.11.0
```

Comment:
numpy [is included](https://github.com/broadinstitute/cms/blob/master/conda-environment.yml#L32) in the conda package list. Do we need to also include it in the pip dependencies?